CONTRIBUTING.md

```diff
@@ -20,11 +20,14 @@ To check out any local changes you've made:
 
 A local server should start and you should be able to access a local version of the website from there.
 
-## Testing changes to book code snippets
+## Testing changes to learning material code examples
 
-The code in the book is compiled and tested to make sure that the examples work for readers.
+The code in the learning materials (e.g., Bevy Book, Quick Start Guide, Advanced Examples, etc.) is compiled, formatted, and tested to make sure that the examples work for readers.
 
-To check your local changes, move into the `code-validation` folder of this project using `cd code-validation`, and then run `cargo test`. If you write a new section of the book, you have to add it in `code-validation/src/lib.rs` to enable code validation for it.
+To validate your code snippets either `./learning-code-examples/validate_examples.sh` (Recommended) or from the root of the project `cd learning-code-examples && cargo check --examples && cargo clippy --examples && cargo fmt --check`.
```

Comment:
Should this also recommend running `validate_examples.sh` instead? It's not cross-platform, but it is shorter.

Reply:
That is what I intended; the `(Recommended)` is intended to apply towards `./learning-code-examples/validate_examples.sh`. I'll experiment with the wording to try and make it less ambiguous.